ios/FluentUI/SegmentedControl/SegmentedControlTokens.swift

```diff
@@ -0,0 +1,129 @@
+        switch style {
+        case .primaryPill:
+            return DynamicColor(light: globalTokens.neutralColors[.grey94], dark: globalTokens.neutralColors[.grey8])
+        case .onBrandPill:
+            return DynamicColor(light: globalTokens.brandColors[.shade20].light, dark: globalTokens.neutralColors[.grey8])
+        }
+    }
+
+    /// Defines the background color of the selected segments of the `SegmentedControl` when disabled.
+    open var disabledSelectedTabColor: DynamicColor {
+        switch style {
+        case .primaryPill:
+            return DynamicColor(light: globalTokens.neutralColors[.grey80], dark: globalTokens.neutralColors[.grey30])
+        case .onBrandPill:
+            return DynamicColor(light: globalTokens.neutralColors[.white], dark: globalTokens.neutralColors[.grey30])
+        }
+    }
+
+    /// Defines the label color of the unselected segments of the `SegmentedControl`.
+    open var restLabelColor: DynamicColor {
+        switch style {
+        case .primaryPill:
+            return DynamicColor(light: aliasTokens.foregroundColors[.neutral3].light, dark: aliasTokens.foregroundColors[.neutral2].dark)
+        case .onBrandPill:
+            return DynamicColor(light: aliasTokens.foregroundColors[.neutralInverted].light, dark: aliasTokens.foregroundColors[.neutral2].dark)
+        }
+    }
+
+    /// Defines the label color of the selected segments of the `SegmentedControl`.
+    open var selectedLabelColor: DynamicColor {
+        switch style {
+        case .primaryPill:
+            return DynamicColor(light: aliasTokens.backgroundColors[.neutral1].light, dark: aliasTokens.foregroundColors[.neutralInverted].dark)
+        case .onBrandPill:
+            return DynamicColor(light: aliasTokens.foregroundColors[.brandRest].light, dark: aliasTokens.foregroundColors[.neutral1].dark)
+        }
+    }
+
+    /// Defines the label color of the unselected segments of the `SegmentedControl` when disabled.
+    open var disabledLabelColor: DynamicColor {
+        switch style {
+        case .primaryPill:
+            return DynamicColor(light: globalTokens.neutralColors[.grey70], dark: globalTokens.neutralColors[.grey26])
+        case .onBrandPill:
+            return DynamicColor(light: globalTokens.brandColors[.shade10].light, dark: globalTokens.neutralColors[.grey26])
+        }
+    }
+
+    /// Defines the label color of the selected segments of the `SegmentedControl` when disabled.
+    open var disabledSelectedLabelColor: DynamicColor {
+        switch style {
+        case .primaryPill:
+            return DynamicColor(light: globalTokens.neutralColors[.grey94], dark: globalTokens.neutralColors[.grey44])
+        case .onBrandPill:
+            return DynamicColor(light: globalTokens.neutralColors[.grey74], dark: globalTokens.neutralColors[.grey44])
+        }
+    }
+
+    /// The color of the unread dot when the `SegmentedControl` is enabled.
+    open var enabledUnreadDotColor: DynamicColor {
```

Comment:
Although the disabledUnreadDotColor is just the same as the label color, I'm wondering if we shouldn't have it here for completeness (and returning the disabledLabelColor).
It will make the disabledUnreadDotColor configurable from the token system instead of hard coded.
The tradeoff is obviously an extra property that will add on the binary size.